Release 0.1.157 - TypeError: Cannot read property 'isBatchingLegacy' of undefined

### Description

Hello, 

Trying this library this morning (which by the way seems pretty amazing), I found out the following error on release 0.1.157.
`TypeError: Cannot read property 'isBatchingLegacy' of undefined`

### Environnement :
Platform : Android
Device : Read device (google pixel 3a)




### Version

0.1.157

### Steps to reproduce

### My code : 
```
const SkiaRaw = observer(({   }: {   }) => {
  
  const image = useImage('https://bit.ly/3fkulX5')
  if (!image) {
    return null
  }
 
  return (
    <View style={{ width: 300, height: 300, backgroundColor: 'white' }}>
      <Canvas style={{ flex: 1 }}>{image && <Image image={image} fit="contain" x={0} y={0} width={256} height={256} />}</Canvas>
    </View>
  )
})
 
export const Skia = SkiaRaw
```

### The error : 
`TypeError: Cannot read property 'isBatchingLegacy' of undefined`

### The workaround : 
Downgrading to 0.1.156 made it work

### Snack, code example, screenshot, or link to a repository

Code shared in **Steps to reproduce** section

I recently faced a similar issue, where I had in package.json 
`"dependencies": { "@shopify/react-native-skia": "^0.1.136", "react": "17.0.2", "react-native": "0.66.5" }` 
and still was not able to install node modules.

I had the following error in the console:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: react-native-skia-charts@0.0.2
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   dev react@"^17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@">=18.0" from @shopify/react-native-skia@0.1.162
npm ERR! node_modules/@shopify/react-native-skia
npm ERR!   dev @shopify/react-native-skia@"^0.1.136" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/vladryabinin/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vladryabinin/.npm/_logs/2022-11-24T18_02_52_111Z-debug-0.log

Process finished with exit code 1

```

In order to fix that I had to remove `^` symbol before the skia version, like this:
 `"@shopify/react-native-skia": "0.1.136",`
 
Hope it'll help somebody,
Cheers!

--- a/docs/docs/getting-started/installation.md
+++ b/docs/docs/getting-started/installation.md
@@ -8,6 +8,7 @@ slug: /getting-started/installation
 React Native Skia brings the [Skia Graphics Library](https://skia.org/) to React Native.
 Skia serves as the graphics engine for Google Chrome and Chrome OS, Android, Flutter, Mozilla Firefox, Firefox OS, and many other products.
 
+> **Version compatibility**: `react-native@>=0.66` and `react@>=18` are required.
 
 ```sh
 yarn add @shopify/react-native-skia
@@ -32,8 +33,6 @@ Below is the app size increase to be expected when adding React Native Skia to y
 Run `pod install` on the `ios/` directory.
 
 ## Android
-
-> **Version compatibility**: `react-native@>=0.66` is required.
 
 Currently, you will need Android NDK to be installed.
 If you have Android Studio installed, make sure `$ANDROID_NDK` is available.

